moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -795,7 +796,10 @@ bool ompl_interface::ModelBasedPlanningContext::solve(double timeout, unsigned i
 
       ob::PlannerTerminationCondition ptc = constructPlannerTerminationCondition(timeout, start);
       registerTerminationCondition(ptc);
-      result = ompl_parallel_plan_.solve(ptc, 1, count, hybridize_) == ompl::base::PlannerStatus::EXACT_SOLUTION;
+      if (ompl_parallel_plan_.solve(ptc, 1, count, hybridize_) == ompl::base::PlannerStatus::EXACT_SOLUTION)
```

Comment:
What about the else case?

Reply:
This has been refactored to:

`result.val = errorCode(ompl_parallel_plan_.solve(ptc, 1, count, hybridize_));`

So there is no need for an if/else anymore.